src/bunfig.zig

```diff
@@ -424,6 +424,12 @@ pub const Bunfig = struct {
                         }
                     }
 
+                    if (_bun.get("backend")) |backend| {
```

Comment:
schema.peechy needs to be updated and then `make api` needs to be run in your terminal

the schema.zig file is auto-generated